### ISSUE TITLE
validate-to-desc-cocina report fails if druid not found in Fedora

### ISSUE THREAD

**Issue by arcadiafalcone** [sul-dlss/dor-services-app#3216]
I'm trying to run the validate-to-desc-cocina report on sdr-deploy, and am finding that the report breaks when it gets to a druid that isn't in Fedora. I tried refreshing the cache for those druids but nothing happens because the druid does not exist. Improved behavior would be to report the error and continue running the report, and/or to remove nonexistent druids from the cache. This is slowing down work on pre-migration remediation of descriptive metadata.

**Comment by arcadiafalcone** [comment]
Druids where this has occurred include druid:wk899rd9024 and druid:zz589rj9944.

**Comment by justinlittman** [comment]
@arcadiafalcone  The code should already handle this; it should just report the druid as missing. Can you `git checkout main` and `git pull` to make sure you have latest? What is the command you're running and what error do you get?

**Comment by arcadiafalcone** [comment]
@justinlittman I did a pull this morning before I ran the report and just double-checked that I'm on the main branch. I'm running `bin/validate-to-desc-cocina -ua`. I was running it in `screen` which won't let me scroll up to see the error. I'm running another report right now (I've been removing the problem druids from the local `druids.txt`) and will grab the error message if it happens again.
